Viper Practice/src/controllers/FruitesViewController.swift

```diff
@@ -22,23 +21,23 @@ class TasksViewController: UIViewController {
 
 }
 
-extension TasksViewController : TaskViewProtocol {
-    func displayTasks(_ tasks: [Fruit]) {
-        self.tasks.append(contentsOf: tasks)
+extension FruitesViewController : FruitesViewProtocol {
```

Comment:
we need to avoid extensions whel possible, add this protocol conformance to the original class

Reply:
fixed